src/passes/expressionSplitter.ts

```diff
@@ -35,6 +40,16 @@ function* expressionGenerator(prefix: string): Generator<string, string, unknown
   }
 }
 
+function getRootIdentifier(node: Identifier | IndexAccess): Identifier | IndexAccess {
```

Comment:
Wouldn't be better to have this function after the class definition to improve readability and keep the style of other passes?

Reply:
Actually in this file functions are both below and above the class. So we will have to fix it one day. Creating a good first issue like that would be a good idea